build-packages/service-worker/build-config/workbox-dev.js

```diff
@@ -13,6 +13,10 @@ module.exports = {
         overrideCracoConfig: ({
             cracoConfig
         }) => {
+            if (!cracoConfig.paths) {
+                Object.assign(cracoConfig, { paths: {} });
```

Comment:
`Object.assign` tends to be slower than just assigning property directly (aka `cracoConfig.paths = {}`)

Do we really need to use it that way?

Reply:
Nope. This is done to pass eslint rule without disabling it.

Reply:
Oh, i see, it's already disabled.
Will fix my code then.